src/main/resources/static/js/MainWindow.js

```diff
@@ -675,567 +774,571 @@ var initial = 1000;
-        },
-        error: function (e) {
-            addToLog(e.responseText);
-        }
-    });
+	$.ajax({
+		type : "GET",
+		url : "thing/single",
+		data : {
+			thingId : parseInt(id, 10),
+			frostUrlString : url
+		},
+		success : function(result) {
+			$("#things").val(result.name + " (" + result.frostId + ")")
+					.trigger("change");
+			getThingStreams(result.frostId, true, streams);
+		},
+		error : function(e) {
+			addToLog(e.responseText);
+		}
+	});
 }
 
-
 function loadStream(stream, div) {
-    var rows = stream.units_of_measurement.length;
-    $(div).find("tbody").empty();
-    for (var i = 0; i < rows; i++) {
-        $(div).find("tbody")
-            .append($("<tr>")
-                .append($("<td>")
-                    .text(stream.units_of_measurement[i].symbol)
-                )
-                .append($("<td>")
-                    .append($("<input>")
-                        .attr("type", "text")
-                        .attr("size", "5")
-                    )
-                )
-            );
-    }
+	var rows = stream.units_of_measurement.length;
+	$(div).find("tbody").empty();
+	for (var i = 0; i < rows; i++) {
+		$(div).find("tbody").append(
+				$("<tr>").append(
+						$("<td>").text(stream.units_of_measurement[i].symbol))
+						.append(
+								$("<td>").append(
+										$("<input>").attr("type", "text").attr(
+												"size", "5"))));
+	}
 }
 
 function getThing() {
-    return JSON.parse($("#things option:selected").attr("data-value"));
+	return JSON.parse($("#things option:selected").attr("data-value"));
 
 }
 
 /**
  * This function is called if a configuration is selected
- * @param e a config
+ * 
+ * @param e
+ *            a config
  */
-$("#configs").on("select2:select", function (e) {
-    var json = JSON.parse($("#configs option:selected").attr("data-value"));
-    if (json) {
-        loadConfig(json.id);
-    }
-    openaccordion("currentConfigAcc");
+$("#configs").on("select2:select", function(e) {
+	var json = JSON.parse($("#configs option:selected").attr("data-value"));
+	if (json) {
+		loadConfig(json.id);
+	}
+	openaccordion("currentConfigAcc");
 });
 
-$("#things").on("select2:select", function (e) {
-    var json = getThing();
-    if (json) {
-        getThingStreams(json.frostId);
-    }
+$("#things").on("select2:select", function(e) {
+	var json = getThing();
+	if (json) {
+		getThingStreams(json.frostId);
+	}
 });
 
-
 /**
- * function performed after choosing a sourcefile. Uploads file (and triggers preview afterwards), and opens next part of the accordion (choice of configuration)
+ * function performed after choosing a sourcefile. Uploads file (and triggers
+ * preview afterwards), and opens next part of the accordion (choice of
+ * configuration)
  */
 function fileConfirmed() {
-    upload();
-    optimzeforsource();
-    openaccordion("chooseConfigAcc");
+	upload();
+	optimzeforsource();
+	openaccordion("chooseConfigAcc");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['openaccordion' is not defined.](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4781409)